gradle/verification-metadata.xml

```diff
@@ -255,231 +260,186 @@
          <artifact name="intellij-core-26.0.0.jar">
-            <sha256 value="8ca48058cddc85f7711a68d0c5bf78a9efe98a3589cb5454e2285510e1e85efb" origin="Generated by Gradle because artifact wasn't signed"/>
-         </artifact>
-      </component>
-      <component group="com.android.tools.external.lombok" name="lombok-ast" version="0.2.3">
-         <artifact name="lombok-ast-0.2.3.jar">
-            <pgp value="3872ed7d5904493d23d78fa2c4c8cb73b1435348"/>
+            <sha256 value="8ca48058cddc85f7711a68d0c5bf78a9efe98a3589cb5454e2285510e1e85efb" origin="Artifact wasn't signed"/>
          </artifact>
       </component>
       <component group="com.android.tools.external.org-jetbrains" name="uast" version="26.0.0">
          <artifact name="uast-26.0.0.jar">
-            <sha256 value="fb41e9d1ab00acd01423c7cbe9a04a0a339271a739a12165af11060cb7754b72" origin="Generated by Gradle because artifact wasn't signed"/>
+            <sha256 value="fb41e9d1ab00acd01423c7cbe9a04a0a339271a739a12165af11060cb7754b72" origin="Artifact wasn't signed"/>
          </artifact>
       </component>
       <component group="com.android.tools.layoutlib" name="layoutlib-api" version="26.0.0">
          <artifact name="layoutlib-api-26.0.0.jar">
-            <sha256 value="2d0e9e0f7f4dfe22f12f84a8f0c4764c43bb8de69ecffcbc3deda29037dedd2b" origin="Generated by Gradle because artifact wasn't signed"/>
+            <sha256 value="2d0e9e0f7f4dfe22f12f84a8f0c4764c43bb8de69ecffcbc3deda29037dedd2b" origin="Artifact wasn't signed"/>
          </artifact>
       </component>
       <component group="com.android.tools.lint" name="lint" version="26.0.0">
          <artifact name="lint-26.0.0.jar">
-            <sha256 value="949a972800e353f51bc9c18410a029ed2b8176639ca307cc0d5c93f36e81dc1f" origin="Generated by Gradle because artifact wasn't signed"/>
+            <sha256 value="949a972800e353f51bc9c18410a029ed2b8176639ca307cc0d5c93f36e81dc1f" origin="Artifact wasn't signed"/>
          </artifact>
       </component>
       <component group="com.android.tools.lint" name="lint-api" version="26.0.0">
          <artifact name="lint-api-26.0.0.jar">
-            <sha256 value="34eaad7db45b3fa6f817daa8c0c74e8c91a80d849b735b231a5fb31c0777c27b" origin="Generated by Gradle because artifact wasn't signed"/>
+            <sha256 value="34eaad7db45b3fa6f817daa8c0c74e8c91a80d849b735b231a5fb31c0777c27b" origin="Artifact wasn't signed"/>
          </artifact>
       </component>
       <component group="com.android.tools.lint" name="lint-checks" version="26.0.0">
          <artifact name="lint-checks-26.0.0.jar">
-            <sha256 value="a0752e3164203444f77f08662e34267320f9435e76de41864a3e8a8e4781043f" origin="Generated by Gradle because artifact wasn't signed"/>
+            <sha256 value="a0752e3164203444f77f08662e34267320f9435e76de41864a3e8a8e4781043f" origin="Artifact wasn't signed"/>
```

Comment:
Two things:
* Current generation will force an `origin` if none is present. I do not plan to change this at this time.
* Will use present tense.

For me the origin adds clarity. Indeed if an artifact fails signature verification, it will have an `ignored-key`, etc ... so maybe `origin` will be mostly the same value based on context. But as a human, a couple words help.
What we should look into is that there is no entry left with a "Generated by Gradle ..." because that's an indication that an added entry was not checked.